versions.yaml

```diff
@@ -18,17 +18,16 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 versions:
-  - dir: '2/2.32'
+  - dir: '2/2.46.2'
     repo: jenkins2
     tags:
-      - 2.32
-      - 2.32.2
+      - 2.46.2
```

Comment:
We should remove the 2.32.x track. So Jenkins now only has one latest track. I'll file a bug to track this.